toothpick-runtime/src/main/java/toothpick/configuration/ReflectionOnConfiguration.java

```diff
@@ -0,0 +1,18 @@
+package toothpick.configuration;
+
+import toothpick.Factory;
+import toothpick.MemberInjector;
+import toothpick.registries.FactoryRegistryLocator;
+import toothpick.registries.MemberInjectorRegistryLocator;
+
+class ReflectionOnConfiguration implements ReflectionConfiguration {
```

Comment:
Maybe we can discuss this namings later.

I'd prefer to use:
IReflectionConfiguration
NoReflectionConfiguration
ReflectionConfiguration

But we don't use I prefix for interfaces right now. We could wait for @stephanenicolas to be back and discuss about it.

I'd prefer not to use Default... As an implementation doesn't know about what is default and say nothing about what it does.